lib/std/process.zig

```diff
@@ -1762,7 +1762,19 @@ pub fn totalSystemMemory() TotalSystemMemoryError!u64 {
                 error.NameTooLong, error.UnknownName => unreachable,
                 else => return error.UnknownTotalSystemMemory,
             };
-            return @as(usize, @intCast(physmem));
+            return @as(u64, @intCast(physmem));
```

Comment:
The type of `physmem` should also be changed to `u64`.

Reply:
Sorry to be asking this but I want to understand this better, why it is ok to replace `c_ulong` with `u64` here. I am not familiar with how `c_ulong` is handled. From what I gather from the internet https://en.wikipedia.org/wiki/C_data_types#Main_types it is expected to be in range of `[0, ULONG_MAX]` are we sure that it is safe to replace it here?

Reply:
Huh, I could've sworn FreeBSD switched its `hw.physmem` to a `uint64_t` at some point, but from looking at the kernel sources just now, it seems not - it remains an `unsigned long`. So no change necessary.